### PR TITLE
Add new example clients to L0_grpc

### PR DESCRIPTION
**Pull request by rmccorm4**

### CODE REVIEW

File: qa/L0_grpc/test.sh [hunk 1]
```diff
@@ -65,6 +65,8 @@ if [[ "$(< /proc/sys/kernel/osrelease)" == *microsoft* ]]; then
     SIMPLE_CUDASHM_CLIENT_PY=${SDKDIR}/python/simple_grpc_cudashm_client.py
     SIMPLE_MODEL_CONTROL_PY=${SDKDIR}/python/simple_grpc_model_control.py
     SIMPLE_REUSE_INFER_OBJECTS_CLIENT_PY=${SDKDIR}/python/reuse_infer_objects_client.py
+    SIMPLE_KEEPALIVE_CLIENT_PY=${SDKDIR}/python/simple_grpc_keepalive_client.py
```
Comment (tanmayv25) [inline qa/L0_grpc/test.sh:68]: See my request to convert these examples to unit tests in the client PR.

Comment (rmccorm4) [reply qa/L0_grpc/test.sh:68]: As we discussed - I think given the generic nature of the channel_args constructor, these will be a good example to keep for reference when users need custom arguments not exposed by Triton. Moving forward, I will be more picky whether new features should be an example or a unit test.

File: qa/L0_grpc/test.sh [hunk 2]
```diff
@@ -242,6 +252,8 @@ for i in \
    $SIMPLE_CUDASHM_CLIENT \
    $SIMPLE_IMAGE_CLIENT \
    $SIMPLE_ENSEMBLE_IMAGE_CLIENT \
+   $SIMPLE_KEEPALIVE_CLIENT_PY \
```
Comment (tanmayv25) [inline qa/L0_grpc/test.sh:255]: $SIMPLE_KEEPALIVE_CLIENT
$SIMPLE_CUSTOM_ARGS_CLIENT

**Review by tanmayv25 — CHANGES_REQUESTED** [review]

File: qa/L0_grpc/test.sh [hunk 3]
```diff
@@ -25,7 +25,8 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
+#REPO_VERSION=${NVIDIA_TRITON_SERVER_VERSION}
+REPO_VERSION="22.04dev"
```
Comment (tanmayv25) [inline qa/L0_grpc/test.sh:29]: Revert this change

File: qa/L0_grpc/test.sh [hunk 4]
```diff
@@ -123,16 +130,18 @@ else
...
 fi
 
-# Add string_dyna_sequence model to repo
-cp -r ${MODELDIR}/simple_dyna_sequence ${MODELDIR}/simple_string_dyna_sequence
-sed -i "s/simple_dyna_sequence/simple_string_dyna_sequence/g" ${MODELDIR}/simple_string_dyna_sequence/config.pbtxt
-sed -i "s/^platform: .*/backend: \"dyna_sequence\"/g" ${MODELDIR}/simple_string_dyna_sequence/config.pbtxt
-sed -i "/CONTROL_SEQUENCE_CORRID/{n;s/data_type:.*/data_type: TYPE_STRING/}" ${MODELDIR}/simple_string_dyna_sequence/config.pbtxt
-rm -f ${MODELDIR}/simple_string_dyna_sequence/1/model.graphdef
-cp ../custom_models/custom_dyna_sequence_int32/1/libtriton_dyna_sequence.so ${MODELDIR}/simple_string_dyna_sequence/1/
+## Add string_dyna_sequence model to repo
```
Comment (tanmayv25) [inline qa/L0_grpc/test.sh:138]: Why was this commented? May be you added these changes into PR by mistake.

**Review by tanmayv25 — CHANGES_REQUESTED** [review]

File: qa/L0_grpc/test.sh (hunk 4 above)
Comment (rmccorm4) [reply qa/L0_grpc/test.sh]: Yes these were local changes added by mistake 👍

**Review by jbkyang-nvi — APPROVED** [review]
